F/FFMPEG/FFMPEG/build_tarballs.jl

```diff
@@ -12,36 +12,47 @@ products = [
     LibraryProduct(["libavdevice", "avdevice"], :libavdevice),
     LibraryProduct(["libavfilter", "avfilter"], :libavfilter),
     LibraryProduct(["libavformat", "avformat"], :libavformat),
-    LibraryProduct(["libavresample", "avresample"], :libavresample),
     LibraryProduct(["libavutil", "avutil"], :libavutil),
     LibraryProduct(["libpostproc", "postproc"], :libpostproc),
     LibraryProduct(["libswresample", "swresample"], :libswresample),
     LibraryProduct(["libswscale", "swscale"], :libswscale),
 ]
 
 # Dependencies that must be installed before this package can be built.
-# TODO: Theora once it's available
+# TODO: Theora once it's available 
 dependencies = [
     HostBuildDependency("YASM_jll"),
     BuildDependency("nv_codec_headers_jll"),
-    Dependency("libass_jll"; compat="0.15.1"),
+    Dependency("libass_jll"; compat = "0.15.1"),
```

Comment:
Can you please remove all the whitespace changes?  It makes review needlessly more complicated.

Reply:
Sure! Sorry about that, forgot to deactivate format on save